### ISSUE TITLE
How to get reproducible results

### ISSUE THREAD

**Issue by chetanambi** [amaiya/ktrain#334]
Hi - I am using ktrain for text classification problem. The issue I am facing is every time I ran the code I am getting different results. Do you have solution to get reproducible results?

**Comment by amaiya** [comment]
Hello:  As far as training sets, you can ensure the same data split is used with the `random_state` parameter.  As far as training, this is more related to `tf.keras` and TensorFlow 2.  You can take a look at [this previous answer](https://github.com/amaiya/ktrain/issues/147#issuecomment-626314957).  [This post](https://stackoverflow.com/questions/60127576/how-do-i-get-reproducible-results-with-tensorflow-2-0) is more recent and says to use `tf.random.set_seed(seed)`, but I haven't tried it.

**Comment by amaiya** [comment]
I've confirmed that invoking this function before each training run will produce (almost) identical results - tested with DistilBert model:

```python
import tensorflow as tf
import numpy as np
import os
import random
def reset_random_seeds(seed=2):
    os.environ['PYTHONHASHSEED']=str(seed)
    tf.random.set_seed(seed)
    np.random.seed(seed)
    random.seed(seed)
```
